slow_query.go

```diff
@@ -11,18 +11,38 @@ import (
 )
 
 var (
-	RegexSqlString = regexp.MustCompile("'[^']+'")
-	RegexNString   = regexp.MustCompile(`([0-9])+`)
-	RegexHasShard  = regexp.MustCompile(`(?i)from\s+(\w+|"\w+")\.`)
+	RegexSqlString  = regexp.MustCompile("('[^']+')") //Everything inside strings
+	RegexHasShard   = regexp.MustCompile(`(?i)from\s+(\w+|"\w+")\.`)
+	RegexIsDateTime = regexp.MustCompile(`'(\d{4}-\d\d-\d\d \d\d:\d\d:\d\d.\d{6})'`)
+	RegexGuidType   = regexp.MustCompile(`(?i)'(\w{3}-\w{8}-\w{4}-\w{4}-\w{4}-\w{12})'`)
```

Comment:
```
'(\w{3}-
```
That can be `[A-Z]{3}` because it will detect things like `USR` or `ACT` but not `123` which `\w` would catch.

Reply:
ok